fix: reveal guide errors (#172)
Basis-Theory/basistheory-integration-guides: collections/guides/use/reveal-card-data-with-elements.md

## Demo Repository

> To better demonstrate this feature, a demo [repository](https://github.com/Basis-Theory-Labs/display-card-example) that you can clone and run on your local machine has been made available.

Reviewer: ```suggestion
To better demonstrate this feature, we made available a [demo repository](https://github.com/Basis-Theory-Labs/display-card-example) that you can clone and run on your local machine.
```
